packages/foreman/foreman/foreman.repo

```diff
@@ -1,13 +1,13 @@
 [foreman]
-name=Foreman nightly
-baseurl=https://yum.theforeman.org/releases/nightly/$DIST/$basearch
+name=Foreman 1.18
+baseurl=https://yum.theforeman.org/releases/1.18/$DIST/$basearch
 enabled=1
-gpgcheck=0
+gpgcheck=1
```

Comment:
We should automate this in the .spec file so we don't have to change this after branching.